style.css

```
@@ -0,0 +1,241 @@
+*{
+    margin:0;
+    padding:0;
+    box-sizing: border-box;
+}
+
+.navbar{
+    display: flex;
+    justify-content: center;
+    background-color: rgba(60, 16, 216, 0.5);
+    position: sticky;
+    top:0;
+}
+
+.navbar ul{
+    display: flex;
+    list-style: none;
+    margin: 20px 0px;
+}
+
+.navbar ul li{
+    font-family: century;
+    font-size: 1.1rem;
+    font-weight: bold;
+}
+
+.navbar ul li a{
+    text-decoration: none;
+    color:white;
+    padding: 8px 25px;
+    transition: all .5s ease;
+}
+
+.navbar ul li a:hover{
+    background-color: white;
+    color: black;
+    box-shadow: 0 0 10px white;
+}
+
+#home h1{
+    color: white;
+}
+
+#home::before{
+    content: "";
+    position: absolute;
+    top:0;
+    right:0;
+    background: url(img/index_back.jpeg) no-repeat center center/cover ;
+    height: 1000px;
+    width: 100%;
+    z-index: -1;
+    opacity: .8;
+    margin-bottom: 0;
+}
+
+.main{
+    border: 1px solid rgb(248, 245, 245);
+    position: absolute;
+    top:30%;
+    right: 10%;
+    left:10%;
+}
+
+.headings{
+    font-family: center;
+    font-size: 3rem;
+    text-align: center;
+    margin-top: 40px 0px;
```

Margin-top only takes one value.

```suggestion
    margin-top: 40px;
```